src/annotator/components/AdderToolbar.tsx

```diff
@@ -71,15 +66,21 @@ function AdderToolbarArrow({ arrowDirection }) {
   );
 }
 
-/**
- * @param {object} props
- *  @param {number} [props.badgeCount]
- *  @param {IconComponent} [props.Icon]
- *  @param {string} props.label
- *  @param {() => void} props.onClick
- *  @param {string|null} props.shortcut
- */
-function ToolbarButton({ badgeCount, Icon, label, onClick, shortcut }) {
+type ToolbarButtonProps = {
+  badgeCount?: number;
+  Icon?: IconComponent;
+  label: string;
+  onClick: () => void;
+  shortcut: string | null;
+};
```

Comment:
Just checking in that we still want to alphabetize props, not put optional props last or anything.

Reply:
I haven't given this much thought. I was planning to mostly preserve the existing ordering when converting code.